modules/installation-azure-limits.adoc

```diff
@@ -184,6 +184,17 @@ ifndef::ash[]
 endif::ash[]
 |The internal load balancer, each of the three control plane machines, and each
 of the three worker machines each use a private IP address.
+
+|Spot VMs
+|
+ifndef::ash[]
```

Comment:
@chinmayi-chandrasekar When looking at the preview build, `Spot VM` and the component description are appearing in the Azure Stack Hub topic. Based on the 'ifndef::ash[ ]', I am presuming that you did not intend for this content to appear in ASH. Should this content appear in Azure only?

For reference: [Azure Stack Hub account limits](https://deploy-preview-39292--osdocs.netlify.app/openshift-enterprise/latest/installing/installing_azure_stack_hub/installing-azure-stack-hub-account.html#installation-azure-limits_installing-azure-stack-hub-account)

Reply:
@mjpytlak Yes, it is not supposed to be there in Azure Stack Hub. I have made the necessary changes. Thanks.